get_id_and_versionid.sh

```diff
@@ -101,6 +107,8 @@ else
 		v2=$(grep PATCHLEVEL /etc/SuSE-release | cut -d' ' -f3)
 		rval=${v1}.$v2
 		;;
+	ocs)
+		rval=`cat /etc/opencloudos-release | cut -d' ' -f4`
```

Comment:
need the break statement `;;`

Also should be `-f2` here as well.

Reply:
fixed.

Reply:
Same here, should read from `opencloudos-stream-release`